tools/vipps-ecom-api-postman-collection.json

```diff
@@ -5,6 +5,153 @@
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+		{
+			"name": "QR",
+			"item": [
+				{
+					"name": "Generate QR Code Short URL",
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {}
+					},
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{access_token}}",
+								"type": "text"
+							},
+							{
+								"key": "Ocp-Apim-Subscription-Key",
+								"value": "{{Ocp-Apim-Subscription-Key}}",
+								"type": "text"
+							},
+							{
+								"key": "Merchant-Serial-Number",
+								"value": "{{merchantSerialNumber}}",
+								"type": "text"
+							},
+							{
+								"key": "Accept",
+								"value": "text/targetUrl",
```

Comment:
change to image/svg+xml
We dont actually want users to use the text/targetUrl, its only for merchants that **has** to make the QR themselves :)

Reply:
Ah, there is two endpoints for this. You could just delete this one, and only have the next one :)

Reply:
Will do